src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,32 @@
+.stopwatch {
+  position: relative;
+
+  display: flex;
+  width: $container-size;
+  height: $container-size;
+  margin: auto;
```

Comment:
You can't use margins to BEM-block, 8-th from checklist:
[BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.